app/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -306,6 +310,7 @@ describe("PageSummary", () => {
     );
     render(<Subject mocks={[mock, mutationMock]} />);
     const rejectButton = await screen.findByTestId("reject-request");
+    await screen.findByText("Approve and End Experiment");
```

Comment:
I added these `findByText` tests to double check the test was actually testing the correct flow.